src/responder/kcm/kcmsrv_cmd.c

```diff
@@ -129,23 +129,27 @@ struct kcm_reqbuf {
     struct kcm_iovec v_msg;
 };
 
+static uint32_t kcm_input_get_payload_len(struct kcm_iovec v)
```

Comment:
I'm sorry I only noticed this now, but did you really intend to pass in the structure value, not a pointer to the structure?

Reply:
Yep, what is the problem you see with this approach?

Reply:
It seems inefficient for no reason.

Reply:
Also non-idiomatic